docs/appendix.rst

```diff
@@ -59,18 +59,14 @@ This documentation uses a few special terms to refer to Python types:
 
    WBEM connection definitions file
    connections file
-      A file maintained by pywbemcli and managed by the
-      :ref:`Connection command group`.
-      The connections file defines a list of named connections, each of which
-      is either the set of parameters for connecting to a real WBEM server, or
-      the set of files for creating a mock WBEM server.
-      The file name of the connections file is
-      ``pywbemcli_connection_definitions.json`` and it is created by pywbemcli
-      in the current directory.
-      Multiple connections files may exist in different directories.
-      This allows connection definitions to be predefined and pywbemcli
-      commands executed against them using only the connection name (via the
-      ``--name`` general option).
+      A file maintained by pywbemcli and managed by the pywbemcli
+      ``connections`` command group.  The file formt is YAML, the name is
+      ``pywbemcli_connections.yaml`` and the file is in the directory in which
+      pywbemcli is executed. Multiple connection files may exist in different
+      directories. The connection file contains a JSON definition of the
+      parameters for each connection with a name for the connection so that
+      connections may be predefined and pywbemcli commands executed against
+      them using only the connection name.
```

Comment:
This text reverts most of the improvements of PR #295. I assume that was unintended. If so, please start from the text in the master version and propose changes based on that.